### ISSUE TITLE
Update request for "Co-host across Channels"

### ISSUE THREAD

**Issue by pmmgithub** [AgoraIO/Docs-Source#1084]
Hi, please help update this guide in 3.x ILS. Since v2.9.0, the Agora SDK relays the media stream of a host from a source channel to a maximum of 64 destination channels. Currently the guide says the maximum of destination channel is 4. 
We will also update the API ref accordingly and provide the latest version to you afterwards. Thanks!

**Comment by pmmgithub** [comment]
Please review and help publish the API ref for 3.x voice, video, and ILS. We updated the description about the maximum destination channels.
[html.zip](https://github.com/AgoraIO/Docs-Source/files/10334860/html.zip)

**Comment by pmmgithub** [comment]
Hi, this update requested was raised by CS but it's pending on engineers and QA, who think they need time to decide whether to deliver this info to users. So I'm closing this issue now. When the decision is finalized I will create a new issue. Sorry for the confusion.
